recipes/pytim/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pytim" %}
+{% set version = "0.5.5" %}
+{% set sha256 = "6c9b4cd137d876b811430b80109f969ead85567bcf2a383be21982cddb606f44" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
```

Comment:
This is not a pure python package (it builds extensions), so `noarch: python` should be removed.